nginx/nginx.conf.template

```diff
@@ -18,6 +18,9 @@ http {
 
         location / {
             proxy_pass http://web:3000/;
+            proxy_http_version 1.1;
```

Comment:
Move all 3 lines to nginx.conf.dev file which is used for local development

Reply:
The mentioned lines are removed from the main nginx template and added to the specified local template as per instruction.